xblock/fields.py

```diff
@@ -829,6 +838,11 @@ def to_string(self, value):
         """String gets serialized and deserialized without quote marks."""
         return self.to_json(value)
 
+    @property
+    def none_to_xml(self):
+        """Returns True to use a XML node for the field and represent None as an attribute."""
+        return True
```

Comment:
Why is this being set on a property instead of class level, when it's set on a class level in `Field`? Curious :)